src/pages/components/ToolBar.qml

```diff
@@ -264,6 +265,10 @@ Column {
                 }
             }
 
+            DragFilter.screenMargin: Theme.paddingMedium
+            onPressed: DragFilter.begin(mouse.x, mouse.y)
+            onCanceled: DragFilter.end()
```

Comment:
Nit (of nits): Different order than IconButton :)

Reply:
Fixed.